src/projecteurapp.cc

```diff
@@ -523,9 +523,7 @@ void ProjecteurApplication::setCurrentCursorPos(const QPoint& pos)
 void ProjecteurApplication::readCommand(QLocalSocket* clientConnection)
 {
   auto it = m_commandConnections.find(clientConnection);
-  if (it == m_commandConnections.end()) {
-    return;
-  }
+  if (it == m_commandConnections.end()) return;
```

Comment:
I know that I am not consistent myself, but I like to put code after if clauses in braces.

Reply:
Ok. I will update that one in next commit.